platform/engineering/frontend/speedcurve/user-guide.md

```diff
@@ -0,0 +1,151 @@
+# Monitoring performance on VA.gov
+
+SpeedCurve provides insight into the interaction between performance and design to help Veteran-facing services (VFS) teams deliver fast and engaging user experiences.
+
+SpeedCurve also provides performance feedback of each app such as page size and how long it takes to load a page, and can show you the impact new features have on the performance of the app.
+
+SpeedCurve's dashboards provide various perspectives and options. VFS teams can define and monitor performance metrics across VA.gov, test performance in the context of continuous deployment, test responsiveness, and analyze assets. VFS teams can also set a performance budget so teams can see and track when the performance of the site lags.
+
+These are some of SpeedCurve features:
+
+- Track dozens of metrics, including custom metrics.
+- Create performance budgets and get alerts.
+- Diagnose performance issues at the page level (Provided with Google's Lighthouse & PageSpeed Insights audits, there is a list of performance rules, stacked and color-coded in their order of importance.)
+- Analyze the performance impact of ads and third parties.
```

Comment:
```suggestion
With SpeedCurve, we can:

- Track dozens of metrics
- Create performance budgets and get alerts
- Diagnose performance issues at the page level (Provided with Google's Lighthouse & PageSpeed Insights audits, there is a list of performance rules, stacked and color-coded in their order of importance)
- Analyze the performance impact of ads and third parties
```
I have a thing about not ending bullet points with periods. Don't mind me. :see_no_evil: